app/views/users/new.html.erb

```diff
@@ -1,52 +1,100 @@
 <% content_for :head do %>
   <%= javascript_include_tag "user" %>
+  <%= javascript_include_tag "auth_providers" %>
 <% end %>
 
-<% content_for :heading_class, "pb-0" %>
-<% content_for :heading do %>
-  <div class='header-illustration new-user-main'>
```

Comment:
I'm surprised to see the header-illustration being removed from the signup pages, since it's one of the few pieces of genuinely original design in our project! But it's more surprising to see it removed from just this page, and not from the other pages that share the same design.

What's the design rationale, if any, behind removing it?

We don't have much in-house capacity for this kind of design, so it concerns me that whenever external companies come in, they often focus on these few pages (login signup etc) and keep redesigning over the top of whoever made the last external design changes. This leads to no redesign being anywhere near comprehensive, and each redesign not quite covering the same (small) ground as the previous one, and we end up with widespread inconsistencies.

Reply:
Rationale for the removal was to make login and signup screens consistently looking and to spare some real estate on the screen. I guess we had not realized the importance of the art work.

Nevertheless, we asked our designer to update the design to incorporate the globe image and tabs similar to tabs found in GPX Traces in OSM.

Reply:
Implemented alternative design with globe image on top of signup/login screens. 
Updated to use bootstrap tabs similar to the existing tabs in OSM website.